tests/api_tests/features/api_tests.feature

```diff
@@ -63,33 +74,34 @@ Feature: Newsfeed API tests
       | field_etc | etc_data   |
     And post request to /newsfeed/123/subscriptions/ is received with body:
       | to_newsfeed_id | 124 |
-    And delete request to /newsfeed/123/subscriptions/4567/ is received
+    And delete request to /newsfeed/123/subscriptions/{subscription_id}/ is received
     Then get get request to /newsfeed/123/events/ is received
     And response status should be 200
     And response should contain the event
 
 
+    # Negative test cases
 
+  Scenario: Add events if there is no data key
+    When post request to /newsfeed/123/events/ is received without data key
+    Then response status should be 400
 
-    # Negative test cases should be discussed
-
-  Scenario: Add events (subscriptions) using incorrect type  - TBD (empty fields, not json, incorrect url ???)
-    When post request to /newsfeed/123/events/ is received with body:
-    Then response status should be 500
-    And response should include the error message - ?
-
-  Scenario: To get deleted events
-    Given post request to /newsfeed/123/events/ is received with body:
-      | field_1   | some_data  |
-      | field_2   | other_data |
-      | field_etc | etc_data   |
-    When delete request to /newsfeed/123/events/4567/ is received
-    Then response status should be 204
-    And this event should be absent
-    And get request to /newsfeed/123/events/ is received
-    And response status should be 500
-    And response should include error message
+  Scenario: Add events with empty data
+    When post request to /newsfeed/123/events/ is received with with empty fields
+    Then response status should be 400
 
+  Scenario: Add events in incorrect format
+    When post request to /newsfeed/123/events/ is received in incorrect type
```

Comment:
Could you please add some test data like `<string>` or `[<string_1>, <string_2>]`?